Formula/olsrd.rb

```diff
@@ -22,13 +22,45 @@ class Olsrd < Formula
 
   depends_on "coreutils" => :build # needs GNU cp
```

Comment:
Can we assume GNU `cp` exists on all Linux?

Reply:
Yes, I think we can.  I've changed this dependency to be macOS-only.